Sources/Testing/Issues/Issue+Recording.swift

```diff
@@ -74,9 +74,31 @@ extension Issue {
     _ comment: Comment? = nil,
     sourceLocation: SourceLocation = #_sourceLocation
   ) -> Self {
-    let sourceContext = SourceContext(backtrace: .current(), sourceLocation: sourceLocation)
-    let issue = Issue(kind: .unconditional, comments: Array(comment), sourceContext: sourceContext)
-    return issue.record()
+    return Self.record(comment, sourceLocation: sourceLocation, severity: .error)
+  }
+
+  /// Record an issue when a running test fails unexpectedly.
+  ///
+  /// - Parameters:
+  ///   - comment: A comment describing the expectation.
+  ///   - sourceLocation: The source location to which the issue should be
+  ///     attributed.
+  ///   - severity: The severity of the issue.
+  ///
+  /// - Returns: The issue that was recorded.
+  ///
+  /// Use this function if, while running a test, an issue occurs that cannot be
+  /// represented as an expectation (using the ``expect(_:_:sourceLocation:)``
+  /// or ``require(_:_:sourceLocation:)-5l63q`` macros.)
+  @_spi(Experimental)
+  @discardableResult public static func record(
+    _ comment: Comment? = nil,
+    sourceLocation: SourceLocation = #_sourceLocation,
+    severity: Severity
```

Comment:
I think `severity:` should be ordered between the `_ comment:`  and `sourceLocation:` parameters. (Same suggestion for the other new overload, below.)

Reply:
(If you take this suggestion, be sure to adjust callers _and_ the DocC documentation atop each method.)